### DISCUSSION TITLE
Able to refer back to a previously selected wildcard in the same prompt?

### DISCUSSION THREAD

**Discussion by chaewai** [adieyal/sd-dynamic-prompts#833] [Ideas]
Would something like this be possible with the current commands? Wasn't able to find it:

Photo with a __color__(1) frame. A man wearing a __color__(1) shirt with __color__(2) hair

i.e. I'd like the colors to be randomly selected, but I want the shirt to refer back to the same color as the randomly rolled frame, and then have a new different color for the hair. Thanks!

**Comment by Hyphona** [comment]
Something like this should work:

`${color={blue|yellow|green}} Photo with a ${color} frame. A man wearing a ${color} shirt with {blue|yellow|green} hair`

[https://github.com/adieyal/sd-dynamic-prompts/blob/main/docs/SYNTAX.md#variables](url)
